undead-discord.css

```diff
@@ -33,6 +33,66 @@
   border-bottom: 1px solid var(--darkest-bg-color);
 }
 
+/* The following 4 blocks color the keybind-shortcut modal and buttons */
+.ddr-arrows { display: none; }
```

Comment:
This still needs to be changed. After this I can merge it.

Reply:
And, if there is only one instance of `display: none`, where exactly do you want me to put that?

Reply:
I have provided a direct link in the previous conversation.

Reply:
Hm, now I feel like a moron xD I've done as asked. Sorry it took so long